modules/packages/vendored/apt_get.mustache

```diff
@@ -32,7 +31,9 @@ apt_get_options = ["-o", "Dpkg::Options::=--force-confold",
                    "-o", "Dpkg::Options::=--force-confdef",
                    "-y"]
 
-if LooseVersion(apt_version) < LooseVersion("1.1"):
+# assume apt-get -v returns a simple n.n or n.n.n version so take the first
+# two numbers and convert to a float
+if float(".".join(apt_version.split(".")[0:2])) < float("1.1"):
```

Comment:
I'd suggest not converting to float here, comparing 2 lists of ints does what you want, I think:

```suggestion
if [int(x) for x in apt_version.split(".")[0:2]] < [1, 1]:
```

Reply:
I'll try it out.

Reply:
I reverted to an even simpler solution, string lexical order. Should work in the only ints case as well as things like a, alpha, beta which I don't expect regardless.